dts/x86/intel/raptor_lake.dtsi

```diff
@@ -161,47 +161,59 @@
 			compatible = "intel,penwell-spi";
 			vendor-id = <0x8086>;
 			device-id = <0x7aaa>;
-			#address-cells = <1>;
-			#size-cells = <0>;
 			pw,cs-mode = <0>;
 			pw,cs-output = <0>;
 			pw,fifo-depth = <64>;
 			cs-gpios = <&gpio_0_i 15 GPIO_ACTIVE_LOW>;
 			clock-frequency = <100000000>;
 			interrupts = <PCIE_IRQ_DETECT IRQ_TYPE_LOWEST_LEVEL_LOW 3>;
 			interrupt-parent = <&intc>;
+			spi_dma0: dma {
+				compatible = "intel,lpss";
+				#dma-cells = <1>;
+				status = "okay";
+			};
+
 			status = "okay";
 		};
 
 		spi1: spi1 {
 			compatible = "intel,penwell-spi";
 			vendor-id = <0x8086>;
 			device-id = <0x7aab>;
-			#address-cells = <1>;
-			#size-cells = <0>;
```

Comment:
Aren't these needed so that you can list connected SPI devices as child-nodes under the SPI controller?